README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://img.shields.io/travis/ReSwift/ReSwift/master.svg?style=flat-square)](https://travis-ci.org/ReSwift/ReSwift) [![Code coverage status](https://img.shields.io/codecov/c/github/ReSwift/ReSwift.svg?style=flat-square)](http://codecov.io/github/ReSwift/ReSwift) [![CocoaPods Compatible](https://img.shields.io/cocoapods/v/ReSwift.svg?style=flat-square)](https://cocoapods.org/pods/ReSwift) [![Platform support](https://img.shields.io/badge/platform-ios%20%7C%20osx%20%7C%20tvos%20%7C%20watchos-lightgrey.svg?style=flat-square)](https://github.com/ReSwift/ReSwift/blob/master/LICENSE.md) [![License MIT](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/ReSwift/ReSwift/blob/master/LICENSE.md) [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg?style=flat-square)](https://houndci.com)
 
-**Supported Swift Versions:** Swift 3.2, 4.0
+**Supported Swift Versions:** Swift 3.2, 4.0, 4.2, 5.0
```

Comment:
```suggestion
**Supported Swift Versions:** Swift 4.0, 4.2, 5.0
```
Since Xcode 10.2 doesn't support Swift 3.2.

Reply:
It is still possible to use ReSwift with older versions of Xcode via CocoaPods, or manual integration.
I will look into dropping Swift 3.2 for our next major release but we should maintain backwards compatibility for the moment.